src/dmd/json.d

```diff
@@ -526,20 +532,20 @@ public:
             for (size_t i = 0; i < s.packages.dim; i++)
             {
                 Identifier pid = (*s.packages)[i];
-                stringPart(pid.toChars());
+                stringPart(pid.toString());
                 buf.writeByte('.');
             }
         }
-        stringPart(s.id.toChars());
+        stringPart(s.id.toString());
         stringEnd();
         comma();
-        property("kind", s.kind());
-        property("comment", s.comment);
+        property("kind", toDString(s.kind()));
+        property("comment", toDString(s.comment));
```

Comment:
Nit: another advantage of making `toDString` a global function is that you can do the more natural: `s.comment.toDString()`